<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssignmentsWidget</class>
 <widget class="QWidget" name="AssignmentsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="assignmentRootLayout">
   <item>
    <widget class="QGroupBox" name="assignmentGroupBox">
     <property name="title">
      <string>Assignments</string>
     </property>
     <layout class="QVBoxLayout" name="assignmentLayout">
      <item>
       <widget class="QListWidget" name="assignmentList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="assignButton">
          <property name="text">
           <string>Assign</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton">
        <property name="text">
         <string>Set Color</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
